mqemitter.js

```diff
@@ -1,61 +1,43 @@
-/*
- * Copyright (c) 2014-2020, Matteo Collina <hello@matteocollina.com>
- *
- * Permission to use, copy, modify, and/or distribute this software for any
- * purpose with or without fee is hereby granted, provided that the above
- * copyright notice and this permission notice appear in all copies.
- *
- * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
- * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
- * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
- * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
- * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
- * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
- * IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
-*/
 'use strict'
 
-var Qlobber = require('qlobber').Qlobber
-var assert = require('assert')
-var fastparallel = require('fastparallel')
+const { Qlobber } = require('qlobber')
+const assert = require('assert')
+const fastparallel = require('fastparallel')
 
 function MQEmitter (opts) {
   if (!(this instanceof MQEmitter)) {
     return new MQEmitter(opts)
   }
 
-  var that = this
+  const that = this
 
   opts = opts || {}
 
-  opts.wildcardOne = opts.wildcardOne || '+'
-  opts.wildcardSome = opts.wildcardSome || '#'
-  opts.separator = opts.separator || '/'
-
   this._messageQueue = []
   this._messageCallbacks = []
   this._parallel = fastparallel({
     results: false,
     released: released
   })
 
-  this.concurrency = opts.concurrency
+  this.concurrency = opts.concurrency || 0
 
-  this.current = 0
+  this._current = 0
```

Comment:
please do not change this.

Reply:
Is the `current` a public variable that used by others?

Reply:
it was identified as public. Changing it is a semver-major change. Moreover it’s useful.